Expo1900/Expo1900/Controller/ExpositionViewController.swift

```diff
@@ -0,0 +1,215 @@
+//
+//  ExpositionViewController.swift
+//  Expo1900
+//
+//  Created by Danny, Gray on 3/13/24.
+//
+
+import UIKit
+
+class ExpositionViewController: UIViewController {
```

Comment:
```suggestion
final class ExpositionViewController: UIViewController {
```

static dispatch를 위해서 final은 디폴트로 붙여주세용